Packs/SecureWorks/Integrations/TaegisXDR/TaegisXDR.py

```diff
@@ -205,14 +262,126 @@ def fetch_alerts_command(client: Client, args=None):
     return results
 
 
-def fetch_investigation_alerts_command(client: Client, args=None):
+def fetch_incidents(client: Client, env: str, args=None):
+    """
+    Fetch Taegis Investigations for the use with "Fetch Incidents"
+    """
+    page_size = args.get("max_fetch", 200)
+    if page_size > 200:
+        raise ValueError("Max Fetch cannot be more then 200")
+
+    include_archived = False
+
+    query = """
+    query investigations(
+          $page: Int,
+          $perPage: Int,
+          $status: [String],
+          $createdAfter: String,
+          $orderByField: OrderFieldInput,
+          $orderDirection: OrderDirectionInput
+      ) {
+          allInvestigations(
+              page: $page,
+              perPage: $perPage,
+              status: $status,
+              createdAfter: $createdAfter,
+              orderByField: $orderByField,
+              orderDirection: $orderDirection
+          ) {
+            id
+            tenant_id
+            description
+            key_findings
+            alerts {
+                id
+                alert_type
+                severity
+                message
+            }
+            archived_at
+            created_at
+            updated_at
+            service_desk_id
+            service_desk_type
+            latest_activity
+            priority
+            status
+            assets {
+                id
+                hostnames {
+                    id
+                    hostname
+                }
+                tags {
+                    tag
+                }
+            }
+        }
+    }
+    """
+
+    variables = {
+        "orderByField": "created_at",
+        "orderDirection": "asc",
+        "page": 0,
+        "perPage": page_size,
+        "status": ["Open", "Active"]
+    }
+
+    last_run = demisto.getLastRun()
+    demisto.debug(f"Last Fetch Incident Run: {last_run}")
+
+    now = datetime.now()
+    start_time = now - timedelta(days=1)  # Default start if first ever run
+    if last_run and "start_time" in last_run:
+        start_time = last_run.get("start_time")
+        variables["createdAfter"] = start_time
```

Comment:
```suggestion
    variables["createdAfter"] = start_time
```

Reply:
when we dont have the "last_run", then the start time should be this set (so we wont get older incidents)